gcp/config.ts

```diff
@@ -0,0 +1,18 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as utils from "./utils";
+
+let config = new pulumi.Config();
+
+export const GcpCloud = "gcp";
+export const LightbendNamespace = "lightbend";
+
+export const cloud = config.get<string>("cloud") || GcpCloud;
+export const zone = config.get<string>("gcp:zone") || "europe-west1-b";
+export const region = config.get<string>("gcp:region") || "europe-west1";
```

Comment:
Don't think there should be a default here either?

Reply:
we can remove setting of `region` altogether ?